ApiDoctor.Console/Program.cs

```diff
@@ -2076,155 +2210,125 @@ private static void WriteHttpSnippetsIntoFile(string tempDir, MethodDefinition[]
-                    Directory.CreateDirectory(sdkLinkDirectory);
-                    // only dump a new file when it does not exist.
-                    var fullFileName = Path.Combine(sdkLinkDirectory, includeSdkFileName);
-                    if (!File.Exists(fullFileName))
-                    {
-                        File.WriteAllText(fullFileName, includeSdkText);
-                    }
-                    break;
+            if (parseStatus == CodeSnippetInsertionState.InsertSnippets)
+            {
+                // Remove HTTP tab if it exists and there are no snippets to add
+                if (string.IsNullOrWhiteSpace(codeSnippets) && originalFileContents[requestStartLine - 1].Contains("#tab/http"))
+                {
+                    updatedFileContents = updatedFileContents.Splice(requestStartLine - 1, 1);
+                    insertionLine--;
                 }
-                case "AdditionalTabInsertion":
-                    /* Add the include link at the specified index */
-                    updatedFileContents = string.IsNullOrEmpty(includeText) ? originalFileContents : FileSplicer(originalFileContents, insertionLine, includeText);
-                    break;
-                default:
-                    //Just return and do not insert a snippet if we can't find an proper place to inject the snippet
-                    return;
-            }
+                updatedFileContents = updatedFileContents.Splice(insertionLine, snippetsTabSectionEndLine - insertionLine);
+                updatedFileContents = FileSplicer(updatedFileContents.ToArray(), insertionLine, codeSnippets);
 
-            /* DUMP THE INJECTIONS*/
-            File.WriteAllLines(method.SourceFile.FullPath, updatedFileContents);
+                // dump the injections
+                File.WriteAllLines(method.SourceFile.FullPath, updatedFileContents);
+            }
 
-            /* DUMP THE CODE SNIPPET FILE */
-            var snippetFileContents = "---\r\ndescription: \"Automatically generated file. DO NOT MODIFY\"\r\n---\r\n" +    //header
-                                      $"\r\n```{codeFenceString}\r\n" +     //code fence
-                                      $"\r\n{codeSnippet}\r\n" +            //generated snippet
-                                      "\r\n```";                            //closing fence
-            var directory = Path.Combine(Directory.GetParent(Path.GetDirectoryName(method.SourceFile.FullPath)).FullName, relativePathSnippetsFolder);
-            Directory.CreateDirectory(directory);//Make sure snippet file directory exists
-            var mdFilePath = Path.Combine(directory, snippetFileName);
-            FancyConsole.WriteLine(FancyConsole.ConsoleSuccessColor, $"Writing snippet to {mdFilePath}");
-            File.WriteAllText(mdFilePath, snippetFileContents);//write snippet to file
+            return; // just return and do not insert a snippet if we can't find a proper place to inject the snippet
```

Comment:
```suggestion
```
void function

Reply:
We want to exit the method at this point and not continue process the file.

Reply:
I agree. But it looks like we're at the end of the function. So this may be unnecessary. Yes?

Reply:
I see. I was looking at old code :roll_eyes: